### DISCUSSION TITLE
AddDumpBox Python script

### DISCUSSION THREAD

**Discussion by WidderGit** [thliebig/openEMS-Project#234] [General]
Hello,
I am attempting to create a "dump box" and I have had no luck figuring out the correct context for Python.
What I really want is to know the H field in the solve domain but as far as I can tell I can only get this at the surface of a rectangular cylinder with a square cross section.  If this is not the case I would appreciate knowing otherwise.

Here is what I have tried so far...
   dump = CSX.AddDump('dump')
   start = [-26,-1.25,4]
   stop = [26,1.25,6.2]
   box = dump.AddBox(start, stop)

   #My best guess properties 1 = H field time domain,  0 = no interpolation, 0 = VTK dump file
   CSX.CSProperties.CSPropDumpBox(1,0,0)

Can anyone tell me the correct Python code to output the H field on the surface of the box?

Thanks
Marc

**Comment by thliebig** [comment]
Have a look at the openEMS docu about the [dump box propery](https://docs.openems.de/python/CSXCAD/CSProperties/CSPropDumpBox.html)

But this is only about storing the data. Looking/displaying them is a bit more involved e.g. using paraview.

**Comment by WidderGit** [comment]
Thanks Thorsten
Marc
